libs/go/libp2p_node/acn/utils.go

```diff
@@ -0,0 +1,129 @@
+/* -*- coding: utf-8 -*-
+* ------------------------------------------------------------------------------
+*
+*   Copyright 2018-2019 Fetch.AI Limited
```

Comment:
should be `2018-2021`

Reply:
in new files, if possible set range correctly

Reply:
done